index.html

```diff
@@ -3,23 +3,44 @@
   <head>
     <meta charset="utf-8" />
     <title>Rock Paper Scissors</title>
+    <link rel="stylesheet" href="./style.css" />
   </head>
 
   <body>
+    <h1>Rock Paper Scissors Game</h1>
+    <div class="flex-container-1">
+      <button class="btn flex-item rock">Rock</button>
+    </div>
+    <div class="flex-container-2">
+      <button class="btn flex-item paper">Paper</button>
+      <button class="btn flex-item scissors">Scissors</button>
+    </div>
+    <div>
+      <div class="cpuScore">
+        <h2>Cpu Score</h2>
+      </div>
+      <div class="playerScore">
+        <h2>Player Score</h2>
+      </div>
+    </div>
+
     <script>
-      /* create variables to store cpu and player wins */
       let cpuWins = 0;
       let playerWins = 0;
-
-      /*create variable to store cpu choice as rock, paper or scissors */
       let cpuChoice;
 
-      /*create variable to choose a random number then store it in a variable */
-      /*change value of cpu choice variable depending on value of random number variable */
-      /*create function to run this code and make a different cpu choice each time function is run */
+      const cpuScore = document.querySelectorAll[0]("h2");
+      let updateCpuScore = function () {
+        score.textContent = cpuWins;
+      };
+
+      const playerScore = document.querySelectorAll[1]("h2");
+      let updatePlayerScore = function () {
+        score.textContent = playerWins;
+      };
+
       function setCpuChoice() {
         let cpuChoiceNum = Math.floor(Math.random() * 3);
-
         if (cpuChoiceNum == 0) {
```

Comment:
whats the reasoning behind using `==`?

Reply:
I thought single = sign was only for assigning a value and == was for comparison? Thinking through it now though actually the only options are 1 or 2 so single = would still return true or false and work in the if statement. Changing now